snmp/zfs-freebsd

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env perl
+
+=head1 DESCRIPTION
+
+This is a SNMP extend for ZFS and FreeBSD for use with LibreNMS.
+
+For more information, see L<https://docs.librenms.org/#Extensions/Applications/#zfs>.
+
+=head1 SWITCHES
+
+=head2 -p
+
+Pretty print the JSON.
+
+=head1 SNMPD SETUP EXAMPLES
+
+    extend zfs /etc/snmp/zfs-freebsd
+
+=cut
+
+#Copyright (c) 2017, Zane C. Bowers-Hadley
```

Comment:
We need a GPLv3 compatible license, I can't tell what license this is from the below.

Reply:
that is the standard 2 clause BSDL

Reply:
https://www.gnu.org/licenses/license-list.en.html#FreeBSD

That with "The views and conclusions contained in the software and documentation are those of the authors and should not be interpreted as representing official policies, either  expressed or implied, of the FreeBSD Project." removed from the end.